incubator/ibm-blockchain-network/templates/composer-pvc.yaml

```diff
@@ -0,0 +1,40 @@
+---
+{{- if not .Values.persistence.storageClass }}
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: {{ template "ibm-blockchain-composer-pvc.name" . }}
```

Comment:
I don't like the naming. We don't you base this on the standard fullname template?

e. g. `{{ template "ibm-blockchain-network.fullname" . }}-composer`

Reply:
Name template used for PVCs because it can be used/shared for subsequent charts.  Is there a mechanism for charts to get the fullname of a PVC?